src/main/org/ovirt/jenkins/plugin/OvirtCloud.java

```diff
@@ -0,0 +1,81 @@
+package main.org.ovirt.jenkins.plugin;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import hudson.model.Label;
+import hudson.slaves.Cloud;
+import hudson.slaves.NodeProvisioner;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+
+public class OvirtCloud extends Cloud {
+
+    private static final Logger LOGGER = Logger.getLogger(OvirtCloud.class.getName());
+
+    private Label label;
+
+    public Label getLabel() { return this.label; }
```

Comment:
Is this part of the jenkins api that we have to implement?

Lets make it clear what the public interface for jenkins, and what our implementation.